rules/integrations/aws/exfiltration_ec2_ebs_snapshot_shared_with_another_account.toml

```diff
@@ -80,9 +81,17 @@ type = "esql"
 
 query = '''
 from logs-aws.cloudtrail-* metadata _id, _version, _index
-| where event.provider == "ec2.amazonaws.com" and event.action == "ModifySnapshotAttribute" and event.outcome == "success"
+| where
+    event.provider == "ec2.amazonaws.com"
+    and event.action == "ModifySnapshotAttribute"
+    and event.outcome == "success"
 | dissect aws.cloudtrail.request_parameters "{%{?snapshotId}=%{snapshotId},%{?attributeType}=%{attributeType},%{?createVolumePermission}={%{operationType}={%{?items}=[{%{?userId}=%{userId}}]}}}"
-| where operationType == "add" and cloud.account.id != userId
```

Comment:
doesn't this query already account for when `userId==all` ? since `cloud.account.id != all`

Reply:
Good point. Let me double check this.

Reply:
@imays11 confirmed this, good catch!
Ill merge the PR in for context changes but will leave the logic as is.